frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -522,6 +522,34 @@ To:
             end,
         })
     end
+    --- @note: Currently, only Kobo implements this quirk
+    if Device:hasEinkScreen() and Device:isKobo() then
+        table.insert(self.menu_items.developer_options.sub_item_table, {
+            text = _("Bypass the MXCFB_WAIT_FOR_* ioctls"),
```

Comment:
A translator's comment might be useful (to the effect of only translate bypass, probably).